webapp/package.json

```diff
@@ -63,5 +63,19 @@
     "**/pouchdb-browser": "^7.2.1",
     "**/moment": "^2.26.2"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.1000.5",
+    "@types/chai": "^4.2.12",
+    "@types/mocha": "^8.0.3",
+    "@types/sinon": "^9.0.5",
+    "chai": "^4.2.0",
+    "karma": "~5.0.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sinon": "^1.0.5",
+    "mocha": "^8.1.3",
+    "sinon": "^9.0.3"
+  }
```

Comment:
We already have most of these dependencies at the root level which is why the `devDependencies` was empty before. I think you'll be able to find a way to use the root dependencies rather than having to have them in the `webapp/package.json` too.

Reply:
Ohhh I see, let me try using the root dependencies and I'll get back on that.

Reply:
I removed dependencies in this file and it's working nice :)